src/librustc_errors/emitter.rs

```diff
@@ -721,7 +736,10 @@ impl EmitterWriter {
             draw_note_separator(&mut buffer, 0, max_line_num_len + 1);
             buffer.append(0, &level.to_string(), Style::HeaderMsg);
             buffer.append(0, ": ", Style::NoStyle);
-            buffer.append(0, msg, Style::NoStyle);
+
+            // The extra 9 ` ` is the padding that's always needed to align to the `note: `.
```

Comment:
How about `"note".len() + 5` (and `"suggestion".len() + 5` further down) to have one magic number instead of two?